x-pack/plugins/security_solution/public/timelines/components/timeline/query_bar/index.tsx

```diff
@@ -221,6 +221,7 @@ export const QueryBarTimeline = memo<QueryBarTimelineComponentProps>(
             isInvalid: false,
             isQuickSelection,
             timelineId,
+            hasRangeChanged: true,
```

Comment:
I am not very clear with the introduction of this new variable. Could you please provide me a small explanation of what is it doing?

Reply:
`dispatchUpdateReduxTime` is dispatching A LOT of actions and some of them, in this case the `updateRange` actions, don't need to fire when the time range hasn't change. Changing the range leads to a change in the timeline model, so that should only happen when the range has actually changed. Otherwise, the timeline shows as `changed` when it's loaded.

Reply:
@logeekal Added a comment here https://github.com/elastic/kibana/pull/169239/commits/c987abaf2458e0ad4129bab981382d827a95cb62

Reply:
Got it.. Thanks.